src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add support for getting updated context after a transition
+ * Colors in transitions are now properly handled in PUML dump
```

Comment:
Actually it should be:

```suggestion
 * New lines in transitions won't result in errors when rendering Mermaid and PUML dumps
 * Transitions with labels and colors will be rendered properly when rendering PUML dumps
```

But the CHANGELOG is about new features, not bug fixes, so it may be removed entirely.

Reply:
indeed